packages/shared/src/contexts/BuyCoresContext.tsx

```diff
@@ -74,16 +101,37 @@ export const BuyCoresContextProvider = ({
           case CheckoutEventNames.CHECKOUT_PAYMENT_SELECTED:
             break;
           case CheckoutEventNames.CHECKOUT_COMPLETED:
+            setActiveStep({
+              step: SCREENS.PROCESSING,
+              providerTransactionId: event.data.transaction_id,
+            });
+
             break;
           // This doesn't exist in the original code
           case 'checkout.warning' as CheckoutEventNames:
             break;
           case CheckoutEventNames.CHECKOUT_ERROR:
             break;
+          case CheckoutEventNames.CHECKOUT_LOADED:
+            isCheckoutOpenRef.current = true;
+            break;
+          case CheckoutEventNames.CHECKOUT_CLOSED:
+            isCheckoutOpenRef.current = false;
+            break;
           default:
             break;
         }
       },
+      checkout: {
+        settings: {
+          displayMode: 'inline',
+          frameTarget: 'checkout-container',
+          frameInitialHeight: 500,
+          frameStyle:
+            'width: 100%; background-color: transparent; border: none;',
+          theme: 'dark',
```

Comment:
Guess we need to ensure here again it works on light/dark mode :)

Reply:
yeah, I'll have to put dark background behind

Reply:
a3dff702fe3ede0e9f2711acf0dc326e3356415f